packages/create-block/lib/index.js

```diff
@@ -45,7 +45,7 @@ program
 		) => {
 			await checkSystemRequirements( engines );
 			try {
-				const defaultValues = getDefaultValues( template );
+				const defaultValues = await getDefaultValues( template );
```

Comment:
The fact that all methods need to be converted to async here is a bit concerning. I'm coming to the conclusion that we should call `async getTemplate( templateName );` first and then pass the object to all other methods. This way we avoid converting all methods to async, plus there is no need to check whether we don't download the template multiple time, or process it. I'm happy to apply all refactoring in the related PR I opened earlier.